### PR TITLE
More UTs and fix regression tests

### PR DESCRIPTION
**Pull request by goran-ethernal**
Adding more UTs to different packages and fixing regression tests by using latest `kurtosis-cdk` changes.

### CODE REVIEW

**Review by rbpol — APPROVED** [review]

File: client/client_test.go [hunk 1]
```diff
@@ -318,3 +318,100 @@ func TestClient_ListOffChainData(t *testing.T) {
...
+			} else {
+				require.NoError(t, err)
+				require.Equal(t, tt.signature, result)
+			}
+		})
+	}
+}
+
+func TestClient_Factory_New(t *testing.T) {
+	url := "http://example.com"
```
Comment (Stefan-Ethernal) [inline client/client_test.go:408]: WDYT?
```suggestion
	t.Parallel()
	url := "http://example.com"
```

File: config/config_test.go [hunk 2]
```diff
@@ -81,6 +89,59 @@ func Test_ConfigFileOverride(t *testing.T) {
...
+	t.Run("invalid keystore file", func(t *testing.T) {
+		t.Parallel()
+
+		cfg := types.KeystoreFileConfig{
+			Path:     "non-existent.keystore",
+			Password: "testonly",
+		}
+
+		key, err := NewKeyFromKeystore(cfg)
+		require.Error(t, err)
```
Comment (Stefan-Ethernal) [inline config/config_test.go:127]: Assert for the specific error (`ErrorContains` or `ErrorIs`)?

**Review by Stefan-Ethernal — APPROVED** [review]
LGTM!

File: client/client_test.go (hunk 1 above)
Comment (goran-ethernal) [reply client/client_test.go:410]: Fixed: https://github.com/0xPolygon/cdk-data-availability/pull/116/commits/a397d2614babae345d4566e4b4f593eaaf339dca

**Pull request merged**